### PR TITLE
feat/#227-add-matomo-tracking-to-overturebio

### PR DESCRIPTION
**Pull request by daniel-cy-lu**
Add Gatsby-Matomo Plugin.[(Link)](https://www.gatsbyjs.com/plugins/gatsby-plugin-matomo/).

Edited Readme. Added node/npm version setup commands.

Matomo dashboard now tracks Overture website.

Screenshot:
Matomo Dashboard after overture website visit:
Screenshot 2023-05-30 at 1 32 08 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,11 +19,12 @@
...
-Overture.bio requires NPM v^8.3.0, to ensure all dependencies are installed correctly
+Overture.bio requires NPM v^8.3.0 and Node v^16.0.0 to ensure all dependencies are installed correctly
 
 ```bash
-# If you need to update your NPM version first, use the following command:
-  npm i -g npm
+# If you need to update your NPM, and Node version first, use the following command:
+  npm i -g npm@8.3.0
+  nvm install 16.0.0
 ```
```
Comment (daniel-cy-lu) [inline README.md:28]: command for changing Node version to 16.

File: gatsby-config.js [hunk 2]
```diff
@@ -171,5 +171,13 @@ module.exports = {
...
       },
     },
+    {
+      resolve: 'gatsby-plugin-matomo',
+      options: {
+        siteId: '74',
+        matomoUrl: 'https://webstats.oicr.on.ca/piwik/',
+        siteUrl: 'https://www.overture.bio/',
+      },
+    },
```
Comment (daniel-cy-lu) [inline gatsby-config.js:181]: Where siteId and matomoUrl can be found once you log into Matomo.

Comment (daniel-cy-lu) [reply gatsby-config.js:181]: When working in Dev. `options` also need to have the following key value pair: `dev:true`. Or `dev` is default `false`. And tracking wouldn't work.

Comment (justincorrigible) [reply gatsby-config.js:181]: 🤔 there must be a way to set these values through environment (i.e. from Netlify), so we don't have to hardcode these here. would you mind looking into that?

File: README.md (hunk 1 above)
Comment (daniel-cy-lu) [reply README.md:28]: I see.. that's why it wasn't there in the first place. I will remove my changes!

**Comment by daniel-cy-lu** [comment]
New Screenshots After making changes, Matomo is still able to track website actions:
Screenshot 2023-05-30 at 3 22 57 PM

**Review by justincorrigible — APPROVED** [review]

**Comment by daniel-cy-lu** [comment]
@justincorrigible Thank Justin :D

**Pull request merged**